recipes/noiseflow/meta.yaml

```
@@ -0,0 +1,81 @@
+{% set org = "shakeflow" %}
+{% set name = "noiseflow" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 6bb97817c0bc9f2e5490e2969233bdfbcb1c6c70f236a5e6d156031a3f686656
+
+build:
+  number: 0
+  script: 
+    - git clone https://github.com/xtensor-stack/xtensor-fftw extern/xtensor-fftw  # [unix]
+    - git clone https://github.com/kfrlib/kfr extern/kfr  # [unix]
+    - export LLVM_OPENMP="1"  # [osx]
+    - export NOISEFLOW_USE_CPP="1"  # [unix]
+    - export MACOSX_DEPLOYMENT_TARGET="10.13"  # [osx]
+    - {{ PYTHON }} -m pip install poetry-core@https://github.com/python-poetry/poetry-core/archive/refs/tags/1.6.1.zip
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  skip: True  # [py<39 or python_impl == 'pypy']
+
+
+requirements:
+  build:
+    - "{{ compiler('cxx') }}"
+    - python                              
+    - poetry-core
+    - setuptools                          
+    - cmake
+    - numpy
+    - pip  
+    - make  # [unix]
+    - ninja  
```

```suggestion
  build:
    - {{ compiler('cxx') }}
    - cmake
    - ninja  
```
Ninja may be used on all platforms. Your python build tools should be listed in requirements/host.